lib/loggers/dev-logger.js

```diff
@@ -16,7 +16,7 @@ class DevLogger {
   }
 
   startDevServer (port) {
-    this.console.log(`Houl dev server is running at http://localhost:${port}`)
+    this.console.log(`Houl dev server is running at:\n Local:http://localhost:${port}\n External:http://${devip}:${port}`)
```

Comment:
I think it would be better not to depends on `devIp` here. Can we update the arguments to receive the external IP address so that the logger will not depend on `devIp`?